src/option.c

```diff
@@ -465,16 +467,16 @@ static int
 coap_option_filter_op(coap_opt_filter_t filter,
 		      uint16_t type,
 		      enum filter_op_t op) {
-  size_t index = 0;
+  size_t lindex = 0;
```

Comment:
Why is this change necessary?

Reply:
index is defined in <strings.h> as included by include/coap/encode.hand so the compiler throws up an unnecessary warning.

src/option.c: In function "coap_option_filter_op":
src/option.c:468: warning: declaration of "index" shadows a global declaration

Reply:
Oh, okay. gcc didn't ;-)